src/scenes/Thanks/components/Title/Title.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+import styles from './Title.module.scss';
+
+const Title = () => {
+  const metadata = { email: 'mail@halo-lab.com' };
```

Comment:
why not just 
```suggestion
  const EMAIL = 'mail@halo-lab.com';
```

Reply:
Oh, it will be replaced with useSiteMetadata hook when portfolio branch will be merged with development. I have created similar data structure.